src/wp-includes/block-template.php

```diff
@@ -358,3 +358,38 @@ function _resolve_template_for_new_post( $wp_query ) {
 		$wp_query->set( 'post_status', 'auto-draft' );
 	}
 }
+
+/**
+ * Register a block template.
+ *
+ * @param string       $template_name  Template name in the form of `plugin_uri//template_name`.
+ * @param array|string $args           Object type or array of object types with which the taxonomy should be associated.
+ * @param array|string $args           {
```

Comment:
There are 2 entries for the same `@args` parameter.

Reply:
Fixed in feaf88d1a3d3740b4e7c1c6ae716ba41e628ab20.